xsd/netex_part_3/part3_fares/netex_usageParameterEligibility_version.xsd

```diff
@@ -315,27 +315,27 @@ Rail transport, Roads and Road transport
 	</xsd:group>
 	<xsd:group name="GroupTicketSizeGroup">
 		<xsd:annotation>
-			<xsd:documentation>Group size elements for GROUP TICKET Group.</xsd:documentation>
+			<xsd:documentation>Group size elements for GROUP TICKET Group. This includes the ticket holder(s).</xsd:documentation>
```

Comment:
```suggestion
			<xsd:documentation>Group size elements for GROUP TICKET Group. This includes the card holder(s).</xsd:documentation>
```

Reply:
The idea is that a ticket may consist of more than one card, I guess? (Or is card something else, e.g., Bahncard or the like giving the right to buy a group ticket?)